README.md

```diff
@@ -38,7 +38,12 @@ To unmount the editor instance, call its `destroy()` method:
 editor.destroy();
 ```
 
-For a complete overview of the API, please refer to the [lib/index.d.ts](https://github.com/ls1intum/Apollon/blob/master/lib/index.d.ts) file.
+For a complete overview of the API, please refer to the [lib/es6/index.d.ts](https://github.com/ls1intum/Apollon/blob/master/lib/index.d.ts) file.
+
+### ESModules and CommonJs
+
+Apollon provides both an ESModules, as well as CommonJS version to be included. 
+The lay in `lib/es6` and `lib/es5` and the correct version should be resolved automatically.
```

Comment:
"The lay" sounds like a typo, please formulate it correctly and mention that deep links need to distinguish the version.

We should check if we can avoid deep links in existing external projects

Reply:
I will have a look at the deep imports in Apollon_standalone and Artemis. I think they should be avoidable. In Artemis they were only used in test files anyways.